src/style.css

```diff
@@ -0,0 +1,24 @@
+body {
+  font-family: Arial;
+  font-size: 13px;
+}
+
+.display-1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin: 0;
+}
+
+.display-2 {
+  font-size: 15px;
+  font-weight: normal;
+  margin: 1em 0;
+}
+
+a {
+  color: green;
+}
+
+div {
+  width: 100;
+}
```

Comment:
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins. Setting width and height for all DIV tags is bad practice